src/Lifecycle.js

```diff
@@ -630,7 +632,13 @@ function _clearStorage() {
         // we'll never make any requests, so can pass a bogus HS URL
         baseUrl: "",
     });
-    return cli.clearStores();
+
+    const clear = async () => {
```

Comment:
I would suggest just marking all of `_clearStorage` async, instead of using this mini nested function.

There is a small issue in `onLoggedOut` which currently calls `_clearStorage().done()`, but `done` is an extra method from the `bluebird` promises library that won't exist from a native promise or async method result.

To resolve this, please also change `onLoggedOut` to be async and use `await _clearStorage()` instead.